docker/bootstrap/CHANGELOG.md

```diff
@@ -56,4 +56,8 @@ List of changes between bootstrap image versions.
 
 ## [14] - 2023-02-21
 ### Changes
-- Update build to golang 1.20.1
\ No newline at end of file
+- Update build to golang 1.20.1
+
+## [16] - 2023-03-24
+### Changes
```

Comment:
not important to the PR, but what happened to 15 here?

Reply:
Since the new automated Golang upgrade feature, the increment represents the bootstrap version instead of a simpler counter that we increase by one. So in reality `14` should have been `15` here.

Reply:
I can open a second PR once this one is merged to update that file.

Reply:
ahh that makes sense!